domains/siama.json

```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "Siam3h",
+        "email": "philosiama@gmail.com"
+      },
+      "record": {
+        "TXT": "hello world"
```

Comment:
Please provide valid records

Reply:
Let me go through the documentation first, I am having trouble setting it up